atlas/packages/app/src/utils/number.ts

```diff
@@ -0,0 +1,15 @@
+export const formatNumber = (num: number): string => {
```

Comment:
If this formats a number by adding a space every 3 non zero decimal digits, then it can be shortened to this:
```javascript
const formatNumber1 = n =>
  n
    .toLocaleString('en-US')
    .split(',')
    .join(' ');
```

Reply:
Ah, I knew there has to be something like this built in but couldn't find it. Awesome, will use that instead

Reply:
Fixed in aa685c0041d3153b7e53e26d975fcf4034b215cb